contentcuration/contentcuration/tests/test_authentication.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+from contentcuration.utils.policies import check_policies, get_latest_policies, POLICIES
+
+from base import BaseTestCase
+
+
+class AuthenticationTestCase(BaseTestCase):
+    def test_authenticate_policy_update(self):
+        """
+        Test that authenticated new users are shown the policies page regardless of what page was requested
+        if they have policies they have not yet agreed to.
+        """
+        channel_id = self.channel.pk
+        base_url = '/channels/{}'.format(channel_id)
+
+        self.channel.viewers.add(self.user)
+        self.client.force_login(self.user)
+
+        assert len(check_policies(self.user)) > 0
+        # ensure that a new user is redirected to policy update after authenticating the first time.
+        response = self.client.get(base_url, follow=True)
+        assert "/policies/update" == response.redirect_chain[-1][0]
+
+    def test_channel_access(self):
```

Comment:
In general, a test should only be testing one aspect -- in this case, let's break this test up into tests on whether it will redirect a user to be logged in, another test to redirect to `/edit`, and another to just `/view`.